client/templates/Components/Form/_input.html.twig

```diff
@@ -7,12 +7,12 @@
                 <span class="govuk-caption-xl">{{ label.caption }}</span>
             {% endif %}
             <label class="govuk-label govuk-label--xl {{ labelClass | default('') }}" for="{{ element.vars.id }}">
-                {{ labelRaw is defined and labelRaw ? label.text | raw : label.text }}
+                {{ labelRaw is defined and labelRaw ? label.text : label.text }}
             </label>
```

Comment:
Is this check necessary if the end result for both cases is using label.text?

Reply:
Good spot!